Tweak CMakeLists.txt to use the defaults for bin and lib directories

diff --git a/CMakeLists.txt b/CMakeLists.txt
index aad45d2..a8f15e3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -78,12 +78,12 @@ endif()
 install(
   TARGETS mpiwrapper
   LIBRARY
-  DESTINATION lib
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}
   )
 
 configure_file(mpiwrapperexec.in mpiwrapperexec @ONLY)
 install(
   FILES "${CMAKE_CURRENT_BINARY_DIR}/mpiwrapperexec"
-  DESTINATION bin
+  DESTINATION ${CMAKE_INSTALL_BINDIR}
   PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
   )
